.vscode/launch.json

```diff
@@ -5,7 +5,23 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "type": "pwa-node",
+            "name": "Launch via NPM",
```

Comment:
They are changes needed to easily run the tests in VSCode in debug-mode with breakpoints. I left them in because they could be helpful to other VSCode users. Should I remove them (then probably the whole file should be removed)

Reply:
Maybe could remove. We could add full launch.json in a separate PR if needed (this `launch.json` is just specific to `broker`). My suggestion would be to add this file to `.gitignore` so that everyone can easily setup their own launch configs.